### DISCUSSION TITLE
Restricting workflows by runner group is confusing

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#12463] [Actions]
<img width="822" alt="image" src="https://user-images.githubusercontent.com/2119212/156914165-12cddd2d-6ac6-49cc-96ae-e86a058f15d2.png">

Workflow access
Control how these runners are used by restricting them to specific workflows. [Learn more](https://docs.github.com/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups#changing-what-workflows-can-access-a-runner-group)

* The learn more link here goes nowhere (dangling link)

<img width="483" alt="image" src="https://user-images.githubusercontent.com/2119212/156914285-9059cf4c-8267-4918-82a6-1ddbe386d651.png">

<img width="590" alt="image" src="https://user-images.githubusercontent.com/2119212/156914328-ba9a463d-3824-485d-a623-cd4c963323b2.png">

Learn more about [managing reusable workflows](https://docs.github.com/actions/learn-github-actions/reusing-workflows)

* This help doesn't seem very helpful -- Not sure what it's trying to say or why this link thinks the contents of the link would be helpful.

I tried entering `.github/workflows/spelling.yml@prerelease`, because it seemed like something I might want. I got this:

<img width="661" alt="image" src="https://user-images.githubusercontent.com/2119212/156914185-788519df-6e68-417b-9e6e-080e90ebe3ce.png">

> Workflow is well-formed but does not resolve to a file in any repository you can access.

**Comment by h2floh** [comment]
Hey @jsoref,

thanks for you feedback. Missing documentation seems to not be published. I am sure this will be fixed soon.

In the meantime can you try to also prepend your full repository path to you workflow file reference like:
`myorg/myrepo/.github/workflows/spelling.yml@prerelease`

I think that should fix it for you.

**Reply by jsoref** [reply]
This works: `check-spelling/check-spelling/.github/workflows/spelling.yml@refs/heads/prerelease`
This doesn't work: `check-spelling/*/.github/workflows/spelling.yml@refs/heads/prerelease`
And certainly, this wouldn't work: `check-spelling/*/.github/workflows/spelling.yml@refs/heads/*`

For more fun, apparently the workflows are `,`-delimited and not space/newline delimited. This is really a bad behavior.

I could _probably_ add a `,` into a branch name. I might even be able to add a space into a branch name.
